projects/cdk/utils/miscellaneous/mark-control-as-touched-and-validate.ts

```diff
@@ -16,7 +19,4 @@ export function markControlAsTouchedAndValidate(control: AbstractControl) {
 
         return;
```

Comment:
Unnecessary `return` now. I wonder if order is important. Maybe we should just remove both `return` so that for every array/group it would start with deep nested controls and after all child controls are processed it would process the group/array itself.

Reply:
You're right, that would be shorter